pkgs/by-name/qu/quick-serve/package.nix

```diff
@@ -0,0 +1,77 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  libxkbcommon,
+  autoPatchelfHook,
+  vulkan-loader,
+  zstd,
+  stdenv,
+  darwin,
+  wayland,
+  gtk3,
+  libGL,
+  wget,
+  tftp-hpa,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "quick-serve";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "joaofl";
+    repo = "quick-serve";
+    rev = "v${version}";
+    hash = "sha256-yO2D3w+aURLX8caFuadzRPdQDlI37ZB4CFzvO4J/zq8=";
+  };
+
+  cargoHash = "sha256-Cogt3eS97CM0Ym0k1C74NfIzNeC3ajIWdTsvwZtRL0U=";
+
+  patches = [./remove-e2e-test-path.patch];
+
+  nativeBuildInputs = [
+    pkg-config
+    autoPatchelfHook
+    wget
```

Comment:
Why do we need wget here? Downloading things doesn't work inside the sandbox.

Reply:
it seems to work for localhost? it's for this test & a couple similar: https://github.com/joaofl/quick-serve/blob/0c654a9ecbff027779dd03565b4f0d284f0c4dce/src/servers/http.rs#L83-L92 checking that the various servers run

Reply:
If it is for tests, then it should go into nativeCheckInputs

Reply:
ah sorry missed that somehow. will have to go back through some of the readmes. fixed now